README.md

```diff
@@ -29,8 +35,12 @@ Now, pmcrypto must to be initialized with openpgp using the init function.
 ```
 import { init } from 'pmcrypto';
 
-init(require('openpgp'));
+init();
```

Comment:
We should maybe remove references to v5 and v6 to the readme, since this example snippet is under "v5" so it's a bit confusing